### ISSUE TITLE
[5.x]: Accessibility Issue - Control Panel Status Color Issue for Live/Disabled

### ISSUE THREAD

**Issue by rogerdawkins** [craftcms/cms#15195]
### Description
As a colour blind user I have always used the 'Use shapes to represent statuses' user preference to view shapes rather than colours for the Control Panel Status icon. Although both Live and Disabled are circles that has never been an issue as Live was a filled circle and Disabled was an empty circle so very clear difference. I'm currently using Craft 5.2 and the Live and Disabled are now both filled circles and they appear the same to me.

I have run the two colours through the Accessibility Colour Contrast checker. For AA conformity it requires a contrast value of 4.5:1 but the two current colours have a contrast ratio of 1.21:1 so far from ideal.

### Expected behavior
Three options:
1. Revert to hollow circle for Disabled (preferred)
2. Use a different shape for Disabled, maybe Triangle pointing down
3. Use filled circles with significantly different contrasting colours

Please consider changing the Disabled icon back to the hollow circle.

### Craft CMS version

5.2

### Installed plugins and versions

-

```[tasklist]
### Tasks
```

**Comment by thupsi** [comment]
Just a heads up: There's a new 'status' column that you can enable in element indexes that has a text label. But I don't see a reason the status icon shapes should not continue to honor the user's preference. That seems to be an oversight.

**Comment by rogerdawkins** [comment]
@thupsi Thanks for the reply. I wasn't aware of the 'Status' column, that's useful and I can use that for now. But column space is always tight so I would preview just the Status icon column but made usable. Also, as P&T are pushing Accessibility so heavily I think they should be using more contrasting colours for Live and Disabled as they are both the same shape. 
The status icon shapes are available and work but the Live and Disabled are both circles so need to use colours with more contrast.

**Comment by thupsi** [comment]
Sorry, I got confused and thought the `disabled` icon was a hollow circle only when the `Use shapes to represent statuses` preference was enabled. But the point remains that, yes, you should not have to only rely on color to differentiate that from the `enabled` icon.

**Comment by brandonkelly** [comment]
This is now resolved for the next release! (#15206) Thanks for reporting.

**Comment by brandonkelly** [comment]
Craft 5.2.1 is out with that change. Thanks again!

**Comment by brandonkelly** [comment]
Realized this also affected inactive users. Fixed in 5.2.4.
